### ISSUE TITLE
Microsoft.JSInterop.JSException: Could not find 'tryInitializeFluxor' ('tryInitializeFluxor' was undefined)

### ISSUE THREAD

**Issue by wadekb** [mrpmorris/Fluxor#184]
I am using Azure AD B2C authentication for the site and it seems due to it.  Auth redirects off to Azure and the error occurs when navigating back.

Is there a different way that I should set Fluxor up where we redirect to auth?

**Comment by mrpmorris** [comment]
Have you remembered to include the javascript reference in your HTML, and have you included a <StoreInitializer/> component in your app.razor?

**Comment by wadekb** [comment]
Yes, the last part of the index.html is -

`    </div>
    <script src="_content/Microsoft.Authentication.WebAssembly.Msal/AuthenticationService.js"></script>
    <script src="_framework/blazor.webassembly.js"></script>
    <script src="_content/Fluxor.Blazor.Web/scripts/index.js"></script>
</body>`

and the first component in app.razor is -

`<Fluxor.Blazor.Web.StoreInitializer />`

Sorry, I should have mentioned that when I removed authentication, it all works, no errors and I get the state displayed successfully in the UI.

I have been using Edge (the Chromium Edge) but I have just tested with Chrome and I get no error with it.

I will see if I can get a basic test project together to replicate the issue.

**Comment by mrpmorris** [comment]
That would be useful. We are using aadb2c at work successfully with Fluxor.

**Comment by wadekb** [comment]
I think this is being caused by caching in Edge.

I created a new app, added Fluxor and it worked, added B2C auth and it still worked.  Then I changed it to run on the port of the app that wasn't working correctly and started getting the error in the new app.  Changed it back to the original port and it worked.  Hard refresh didn't do anything but after clearing the cache, seems to fix things.

I get still get weird behaviour when running the different projects on the same port (not running at the same time) ie. after redirecting back from B2C auth, the title of the browser tab displays the name of the other app and doesn't work until after clearing the cache.
